validator/accounts/v2/cmd_accounts.go

```diff
@@ -116,5 +118,25 @@ this command outputs a deposit data string which is required to become a validat
 				return nil
 			},
 		},
+		{
+			Name:        "voluntary-exit",
+			Description: "Performs a voluntary exit on selected accounts",
+			Flags: []cli.Flag{
+				flags.WalletDirFlag,
+				flags.WalletPasswordFileFlag,
+				flags.AccountPasswordFileFlag,
+				flags.VoluntaryExitPublicKeysFlag,
+				featureconfig.AltonaTestnet,
+				featureconfig.OnyxTestnet,
+			},
+			Action: func(cliCtx *cli.Context) error {
+				featureconfig.ConfigureValidator(cliCtx)
+				// if err := ExitAccounts(cliCtx, os.Stdin); err != nil {
```

Comment:
What is this?

Reply:
I commented out the call to the real version because the functionality is not fully implemented yet. Do you want me to remove this?

Reply:
Yeah. No commented code please